examples/src/examples-ssr.js

```diff
@@ -22,4 +22,4 @@ export default function() {
             </body>
         </html>
     `;
-};
+}
```

Comment:
I think it's the difference between a function expression and a function declaration. Or something.

- `function name () {}` has no semi-colon

- `const name = function () {};` does have a semi-colon